.github/workflows/main.yml

```diff
@@ -17,7 +17,14 @@ jobs:
         go-version: 1.16
 
     - name: Install ginkgo
-      run: sudo apt-get install golang-ginkgo-dev
+      run: go get github.com/onsi/ginkgo/ginkgo@v1.16.5
```

Comment:
Good to open an issue to update this in other workflow ymls too.

Reply:
Opened Issue https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/issues/327 to track this one.